lib/src/components/media-card.ts

```diff
@@ -0,0 +1,80 @@
+import '../globals';
+
+const { LitElement, html, css } = window.Lit;
+
+class MediaCard extends LitElement {
+    static styles = css`
+        
+        .wrapper {
+            display: flex;
+            justify-content: center;
+            flex-wrap: wrap;
+            margin-top: 6.25rem;
+        }
+
+        .container {
+            background: white;
+            width: 19.18rem;
+            height: 24rem;
+            margin: 1rem;
+            border: 0.25rem solid black;
+            border-radius: 1rem;
+        }
+
+        .container-image {
+            height: 12rem;
+            margin-bottom: 1rem;
+            background-size: cover;
+            border-radius: 0.75rem 0.75rem 0 0;
+        }
+
+        .container-content {
+            display: flex;
```

Comment:
Tell me what `display: flex` is, and how the subsequent rulesets effect your content.

Reply:
Creates a Flex container and the rulesets control the layout of the container.